.dockerignore

```diff
@@ -0,0 +1,12 @@
+.env
+.env.j2
+.env.template
+.env.test
+.gitignore
+todoapp.service
+dockerfile
+README.md
+ansible/inventory
+ansible/playbook.yml
```

Comment:
Would replacing these two with just `ansible` make more sense, so that if you add anything additional to that folder later they'll automatically be captured?